src/lib/main/main-bot.js

```diff
@@ -0,0 +1,36 @@
+const { askToReview } = require("../review");
+const { addLabels } = require("../label");
+const { isCIRequired } = require("../ci");
+
+async function run(context) {
+  if (
+    context.payload.issue.labels.find(
```

Comment:
shouldn't we `await` for it?

Reply:
No it does not required here, it is working as expected without it.